glusterd2/commands/volumes/commands.go

```diff
@@ -136,12 +136,6 @@ func (c *Command) Routes() route.Routes {
 			Version:     1,
 			RequestType: utils.GetTypeString((*api.VolStatedumpReq)(nil)),
 			HandlerFunc: volumeStatedumpHandler},
-		route.Route{
```

Comment:
need to regenerate the `Endpoints` document.

Reply:
ok